app/models/form_submission_attempt.rb

```diff
@@ -80,6 +81,41 @@ def log_status_change
 
   private
 
+  def simple_forms_api_email(log_info)
+    Rails.logger.info('Preparing to send Form Submission Attempt error email', log_info)
+    simple_forms_enqueue_result_email(:error)
+  end
+
+  def queue_form526_form4142_email(form526_submission_id, log_info)
+    Rails.logger.info('Queuing Form526:Form4142 failure email to VaNotify',
+                      log_info.merge({ form526_submission_id: }))
+    jid = EVSS::DisabilityCompensationForm::Form4142DocumentUploadFailureEmail.perform_async(
+      form526_submission_id
+    )
+    Rails.logger.info('Queuing Form526:Form4142 failure email to VaNotify completed',
+                      log_info.merge({ jid:, form526_submission_id: }))
+  end
+
+  def form526_form4142_email(log_info)
+    if Flipper.enabled?(CentralMail::SubmitForm4142Job::POLLED_FAILURE_EMAIL)
+      queue_form526_form4142_email(Form526Submission.find_by(saved_claim_id:).id, log_info)
```

Comment:
I'm not sure `saved_claim_id` can be directly call on the model; would have to access the `saved_claim` instance and then get its id.

Reply:
Hmm it works in the test cases 🤔 But will double check

Reply:
```ruby
class TestingSavedClaimId < FormSubmissionAttempt
  def check_it
    saved_claim_id
  end
end

irb(main):018> x = TestingSavedClaimId.new
=> #<TestingSavedClaimId id: nil, form_submission_id: nil, aasm_state: "pending", encrypted_kms_key: nil, created_at: nil, updated_at: nil, benefits_intake_uuid: n...
irb(main):019> x.saved_claim=SavedClaim.last
2024-10-28 12:35:27.168479 D [37466:11520 log_subscriber.rb:168] ActiveRecord::Base --   SavedClaim Load (4.1ms)  SELECT "saved_claims"."id", "saved_claims"."created_at", "saved_claims"."updated_at", "saved_claims"."form_id", "saved_claims"."guid", "saved_claims"."type", "saved_claims"."form_ciphertext", "saved_claims"."encrypted_kms_key", "saved_claims"."uploaded_forms", "saved_claims"."form_start_date", "saved_claims"."delete_date", "saved_claims"."metadata", "saved_claims"."metadata_updated_at" FROM "saved_claims" ORDER BY "saved_claims"."id" DESC LIMIT $1  [["LIMIT", 1]]
=> #<SavedClaim::DisabilityCompensation::Form526AllClaim id: 6, created_at: "2024-09-26 12:43:01.935747000 +0000", updated_at: "2024-09-26 12:43:01.935747000 +0000...
irb(main):020> 
irb(main):021> 
irb(main):022> x.check_it
=> 6

```

Looks like it works?

Reply:
Now I know.